MantidQt/CustomInterfaces/src/DynamicPDF/SliceSelector.cpp

```diff
@@ -129,14 +173,101 @@ void SliceSelector::launchBackgroundRemover() {
   //}
   // m_BackgroundRemover->refreshSlice(m_loadedWorkspace,
   // m_selectedWorkspaceIndex);
-  g_log.error("Not implemented...yet");
+  auto title = QString::fromStdString(this->name());
+  auto error = QString::fromStdString("Not so fast, cowboy! (not implemented)");
```

Comment:
Interesting warning  message when you click `Launch Background Remover` button.. :joy_cat: